src/firefly/js/Firefly.js

```diff
@@ -131,6 +131,10 @@ const defFireflyOptions = {
         {id: 'vocat'},
         {id: 'nedcat'}
     ],
+    theme: {
+        customized: undefined,          // a function that returns a customized theme
+        colorMode: undefined,           // dark/light mode.  Default to device's mode.
```

Comment:
You should add a couple of commented out lines for the next few weeks. That way it is quick to change mode.

```js
        // colorMode: () => 'dark',           // dark mode
        // colorMode: () => 'light',          // light mode
```

Reply:
colorMode is a string 'dark' or 'light'.  I will clarify in the existing comment.